### ISSUE TITLE
Extension will still try to zip 1 file even when I told it not to

### ISSUE THREAD

**Issue by Poketostorm** [leoding86/webextension-pixiv-toolkit#122]
**Describe the bug**
"Pack downloaded file(s) to a zip file" is ON and "Alway pack image(s) into zip file even if there is only one image in the illustration" is OFF. When there is only 1 illustration and I click "DL Image" (not "DL Page 1"), it still zips the single illustration, despite the fact that I explicitly told it not to pack singular images into a zip file.

**Expected behavior**
Previously (until today), the Toolkit downloaded single images as image files, and multiple images as zips.

**Example work from Pixiv**
If applicable, add work url to help me to reproduce the problem.

**Screenshots**

**Browser and Version (please complete the following information):**
 - e.g. chrome, edge，firefox
 Chrome

**Settings of extension**
You can export the settings of extension in setting page.
{"MangaPageNumberStartWithOne":false,"alwaysPack":false,"animationJsonFormat":1,"askDownloadSavedWork":true,"autoActivateDownloadPanel":false,"displayWorkTypeLabel":true,"downloadPackFiles":true,"downloadRelativeLocation":null,"downloadSaveAs":false,"downloadTasksWhenDownloadingImages":3,"enableExtTakeOverDownloads":false,"enableExtend":false,"enableExtension":true,"enablePackUgoiraFramesInfo":true,"enablePtkSearch":true,"enableSaveDownloadHistory":1,"enableSaveVisitHistory":true,"enableWhenUnderSeconds":1,"extendDuration":3,"featureKnown":false,"guideShowed":true,"illustrationImageRenameFormat":"{id}_p{pageNum} ({author})","illustrationKeepPageNumber":false,"illustrationPageNumberLength":0,"illustrationPageNumberStartWithOne":false,"illustrationRelativeLocation":"","illustrationRenameFormat":"{id} ({author})","importantNoticeDisplayed":true,"language":"default","mangaImageRenameFormat":"","mangaPageNumberLength":0,"mangaPagesInChunk":99,"mangaRelativeLocation":"","mangaRenameFormat":"","maxDownloadRecords":10000,"maxHistoryItems":10000,"notSaveNSFWWorkInHistory":false,"novelIncludeDescription":false,"novelRelativeLocation":"","novelRenameFormat":"","showUpdateChangeLog":true,"statIllustDownloaded":15879,"statMangaDownloaded":315,"statNovelDownloaded":26,"statUgoiraDownloaded":151,"ugoiraConvertTool":"default","ugoiraCustomFFmpegCommand":"","ugoiraQuanlity":10,"ugoiraRelativeLocation":"","ugoiraRenameFormat":"","version":"5.0.2","visitHistoryType":"list","workCoverSize":1}

**Additional context**

**Comment by Poketostorm** [comment]
Also, my current workaround is to set "Pack downloaded file(s) to a zip file" to OFF while I download single images, but obviously this becomes a pain if I have to keep toggling this setting whenever I encounter a single or multi-work illustration.

**Comment by Pxartist** [comment]
I can confirm that it does indeed attempt to make a .zip file even when specified not do and there is only 1 image.

Nowadays I just left click on the image to enlarge it and then do a Save As...

**Comment by siroroe** [comment]
actually i think itself it's a bug

**Comment by Poketostorm** [comment]
Additionally, with the same settings, it will actually fail to pack certain multi-work illustrations and try to save them one-by-one - on top of that, the filenames are just #.jpg (0, 1, 2, etc.).
For example: https://www.pixiv.net/en/artworks/92047092
I found that this tends to happen on works that had issues being recognized as proper illustrations by the tool, if I recall correctly. The illustration I linked above was by postcard, and in **previous versions** of the Toolkit, it has failed to properly apply my naming scheme (e.g: {id}_p# instead of whatever I set illustrations to). Perhaps this has something to do with that as well?
